# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
#
msgid ""
msgstr ""
"Project-Id-Version: \n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2016-06-27 15:00+0000\n"
"PO-Revision-Date: 2017-02-17 14:09+0100\n"
"Language: da_DK\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Last-Translator: Kræn Hansen <kraen@socialsquare.dk>\n"
"Language-Team: \n"
"X-Generator: Poedit 1.5.4\n"

#: venv/lib/python3.5/site-packages/allauth/account/adapter.py:45
msgid "Usernames can only contain letters, digits and @/./+/-/_."
msgstr "Brugernavn kan kun indeholde bogstaver, tal og  @/./+/-/_."

#: venv/lib/python3.5/site-packages/allauth/account/adapter.py:47
msgid "Username can not be used. Please use other username."
msgstr "Brugernavnet kan ikke bruges. Vælg venligst et andet."

#: venv/lib/python3.5/site-packages/allauth/account/adapter.py:49
msgid "This username is already taken. Please choose another."
msgstr "Dette brugernavn er allerede taget. Vælg venligst et andet."

#: venv/lib/python3.5/site-packages/allauth/account/adapter.py:51
msgid "Too many failed login attempts. Try again later."
msgstr "For mange log ind forsøg. Prøv igen senere."

#: venv/lib/python3.5/site-packages/allauth/account/adapter.py:279
#, python-brace-format
msgid "Password must be a minimum of {0} characters."
msgstr "En adgangskode skal have minimum {0} tegn."

#: venv/lib/python3.5/site-packages/allauth/account/apps.py:8
msgid "Accounts"
msgstr "Konti"

#: venv/lib/python3.5/site-packages/allauth/account/forms.py:53
#: venv/lib/python3.5/site-packages/allauth/account/forms.py:290
#: venv/lib/python3.5/site-packages/allauth/account/forms.py:392
msgid "Password"
msgstr "Adgangskode"

#: venv/lib/python3.5/site-packages/allauth/account/forms.py:54
msgid "Remember Me"
msgstr "Husk mig"

#: venv/lib/python3.5/site-packages/allauth/account/forms.py:60
msgid "This account is currently inactive."
msgstr "Denne konto er i øjeblikket inaktiv."

#: venv/lib/python3.5/site-packages/allauth/account/forms.py:63
msgid "The e-mail address and/or password you specified are not correct."
msgstr "Email og/eller adgangskode du har skrevet er ikke korrekt."

#: venv/lib/python3.5/site-packages/allauth/account/forms.py:66
msgid "The username and/or password you specified are not correct."
msgstr "Brugernavn og/eller adgangskode du har skrevet er ikke korrekt."

#: venv/lib/python3.5/site-packages/allauth/account/forms.py:69
msgid "The login and/or password you specified are not correct."
msgstr "Login og/eller adgangskode du har skrevet er ikke korrekt."

#: venv/lib/python3.5/site-packages/allauth/account/forms.py:78
#: venv/lib/python3.5/site-packages/allauth/account/forms.py:219
#: venv/lib/python3.5/site-packages/allauth/account/forms.py:337
#: venv/lib/python3.5/site-packages/allauth/account/forms.py:416
msgid "E-mail address"
msgstr "Emailadresse"

#: venv/lib/python3.5/site-packages/allauth/account/forms.py:80
#: venv/lib/python3.5/site-packages/allauth/account/forms.py:237
#: venv/lib/python3.5/site-packages/allauth/account/forms.py:332
#: venv/lib/python3.5/site-packages/allauth/account/forms.py:411
msgid "E-mail"
msgstr "Email"

#: venv/lib/python3.5/site-packages/allauth/account/forms.py:85
#: venv/lib/python3.5/site-packages/allauth/account/forms.py:88
#: venv/lib/python3.5/site-packages/allauth/account/forms.py:211
#: venv/lib/python3.5/site-packages/allauth/account/forms.py:215
msgid "Username"
msgstr "Brugernavn"

#: venv/lib/python3.5/site-packages/allauth/account/forms.py:95
msgid "Username or e-mail"
msgstr "Brugernavn eller email"

#: venv/lib/python3.5/site-packages/allauth/account/forms.py:98
msgctxt "field label"
msgid "Login"
msgstr "Log ind [ field label ]"

#: venv/lib/python3.5/site-packages/allauth/account/forms.py:240
msgid "E-mail (optional)"
msgstr "Email (valgfri)"

#: venv/lib/python3.5/site-packages/allauth/account/forms.py:272
msgid "A user is already registered with this e-mail address."
msgstr "En bruger med denne email er allerede oprettet."

#: venv/lib/python3.5/site-packages/allauth/account/forms.py:291
#: venv/lib/python3.5/site-packages/allauth/account/forms.py:393
msgid "Password (again)"
msgstr "Adgangskode (igen)"

#: venv/lib/python3.5/site-packages/allauth/account/forms.py:308
#: venv/lib/python3.5/site-packages/allauth/account/forms.py:382
#: venv/lib/python3.5/site-packages/allauth/account/forms.py:400
#: venv/lib/python3.5/site-packages/allauth/account/forms.py:487
msgid "You must type the same password each time."
msgstr "Du skal indtaste den samme adgangskode hver gang."

#: venv/lib/python3.5/site-packages/allauth/account/forms.py:343
msgid "This e-mail address is already associated with this account."
msgstr "Denne email er allerede associeret med denne konto."

#: venv/lib/python3.5/site-packages/allauth/account/forms.py:345
msgid "This e-mail address is already associated with another account."
msgstr "Denne email er allerede associeret med en anden konto."

#: venv/lib/python3.5/site-packages/allauth/account/forms.py:367
msgid "Current Password"
msgstr "Nuværende adgangskode"

#: venv/lib/python3.5/site-packages/allauth/account/forms.py:368
#: venv/lib/python3.5/site-packages/allauth/account/forms.py:473
msgid "New Password"
msgstr "Ny adgangskode"

#: venv/lib/python3.5/site-packages/allauth/account/forms.py:369
#: venv/lib/python3.5/site-packages/allauth/account/forms.py:474
msgid "New Password (again)"
msgstr "Ny adgangskode (igen)"

#: venv/lib/python3.5/site-packages/allauth/account/forms.py:373
msgid "Please type your current password."
msgstr "Indtast venligst din nuværende adgangskode."

#: venv/lib/python3.5/site-packages/allauth/account/forms.py:425
msgid "The e-mail address is not assigned to any user account"
msgstr "Denne emailadresse tilhører ikke nogen brugerkonto."

#: venv/lib/python3.5/site-packages/allauth/account/forms.py:504
msgid "The password reset token was invalid."
msgstr "Adgangskode nulstillingsnøgle var ikke gyldig."

#: venv/lib/python3.5/site-packages/allauth/account/models.py:25
msgid "user"
msgstr "bruger"

#: venv/lib/python3.5/site-packages/allauth/account/models.py:28
#: venv/lib/python3.5/site-packages/allauth/account/models.py:84
msgid "e-mail address"
msgstr "emailadresse"

#: venv/lib/python3.5/site-packages/allauth/account/models.py:29
msgid "verified"
msgstr "verificeret"

#: venv/lib/python3.5/site-packages/allauth/account/models.py:30
msgid "primary"
msgstr "primær"

#: venv/lib/python3.5/site-packages/allauth/account/models.py:35
msgid "email address"
msgstr "emailadresse"

#: venv/lib/python3.5/site-packages/allauth/account/models.py:36
msgid "email addresses"
msgstr "emailadresser"

#: venv/lib/python3.5/site-packages/allauth/account/models.py:85
msgid "created"
msgstr "oprettet"

#: venv/lib/python3.5/site-packages/allauth/account/models.py:87
msgid "sent"
msgstr "sent"

#: venv/lib/python3.5/site-packages/allauth/account/models.py:88
#: venv/lib/python3.5/site-packages/allauth/socialaccount/models.py:59
msgid "key"
msgstr "nøgle"

#: venv/lib/python3.5/site-packages/allauth/account/models.py:93
msgid "email confirmation"
msgstr "email bekræftigelse"

#: venv/lib/python3.5/site-packages/allauth/account/models.py:94
msgid "email confirmations"
msgstr "email bekræftigelser"

#: venv/lib/python3.5/site-packages/allauth/socialaccount/adapter.py:121
msgid "Your account has no password set up."
msgstr "Din konto har ikke nogen adgangskode."

#: venv/lib/python3.5/site-packages/allauth/socialaccount/adapter.py:128
msgid "Your account has no verified e-mail address."
msgstr "Din konto har ikke nogen verificeret emailadresse."

#: venv/lib/python3.5/site-packages/allauth/socialaccount/apps.py:8
msgid "Social Accounts"
msgstr "Sociale konti"

#: venv/lib/python3.5/site-packages/allauth/socialaccount/forms.py:41
#, python-format
msgid ""
"An account already exists with this e-mail address. Please sign in to that "
"account first, then connect your %s account."
msgstr ""
"En konto med denne emailadresse eksisterer allerede. Log venligst ind med "
"den konto først og forbind så din %s konto."

#: venv/lib/python3.5/site-packages/allauth/socialaccount/models.py:47
#: venv/lib/python3.5/site-packages/allauth/socialaccount/models.py:80
msgid "provider"
msgstr "udbyder"

#: venv/lib/python3.5/site-packages/allauth/socialaccount/models.py:50
msgid "name"
msgstr "navn"

#: venv/lib/python3.5/site-packages/allauth/socialaccount/models.py:52
msgid "client id"
msgstr "klient id"

#: venv/lib/python3.5/site-packages/allauth/socialaccount/models.py:54
msgid "App ID, or consumer key"
msgstr "App id, eller bruger nøgle"

#: venv/lib/python3.5/site-packages/allauth/socialaccount/models.py:55
msgid "secret key"
msgstr "hemmelig nøgle"

#: venv/lib/python3.5/site-packages/allauth/socialaccount/models.py:57
msgid "API secret, client secret, or consumer secret"
msgstr ""

#: venv/lib/python3.5/site-packages/allauth/socialaccount/models.py:62
msgid "Key"
msgstr "Nøgle"

#: venv/lib/python3.5/site-packages/allauth/socialaccount/models.py:70
msgid "social application"
msgstr "social app"

#: venv/lib/python3.5/site-packages/allauth/socialaccount/models.py:71
msgid "social applications"
msgstr "sociale apps"

#: venv/lib/python3.5/site-packages/allauth/socialaccount/models.py:99
msgid "uid"
msgstr "uid"

#: venv/lib/python3.5/site-packages/allauth/socialaccount/models.py:101
msgid "last login"
msgstr "seneste log ind"

#: venv/lib/python3.5/site-packages/allauth/socialaccount/models.py:103
msgid "date joined"
msgstr "tilmeldingsdato"

#: venv/lib/python3.5/site-packages/allauth/socialaccount/models.py:105
msgid "extra data"
msgstr "ekstra data"

#: venv/lib/python3.5/site-packages/allauth/socialaccount/models.py:109
msgid "social account"
msgstr "social konto"

#: venv/lib/python3.5/site-packages/allauth/socialaccount/models.py:110
msgid "social accounts"
msgstr "sociale konti"

#: venv/lib/python3.5/site-packages/allauth/socialaccount/models.py:136
msgid "token"
msgstr ""

#: venv/lib/python3.5/site-packages/allauth/socialaccount/models.py:138
msgid "\"oauth_token\" (OAuth1) or access token (OAuth2)"
msgstr ""

#: venv/lib/python3.5/site-packages/allauth/socialaccount/models.py:141
msgid "token secret"
msgstr ""

#: venv/lib/python3.5/site-packages/allauth/socialaccount/models.py:143
msgid "\"oauth_token_secret\" (OAuth1) or refresh token (OAuth2)"
msgstr ""

#: venv/lib/python3.5/site-packages/allauth/socialaccount/models.py:145
msgid "expires at"
msgstr "udløber den"

#: venv/lib/python3.5/site-packages/allauth/socialaccount/models.py:149
msgid "social application token"
msgstr ""

#: venv/lib/python3.5/site-packages/allauth/socialaccount/models.py:150
msgid "social application tokens"
msgstr ""

#: venv/lib/python3.5/site-packages/allauth/socialaccount/providers/oauth/client.py:77
#, python-format
msgid "Invalid response while obtaining request token from \"%s\"."
msgstr ""

#: venv/lib/python3.5/site-packages/allauth/socialaccount/providers/oauth/client.py:103
#, python-format
msgid "Invalid response while obtaining access token from \"%s\"."
msgstr ""

#: venv/lib/python3.5/site-packages/allauth/socialaccount/providers/oauth/client.py:119
#, python-format
msgid "No request token saved for \"%s\"."
msgstr ""

#: venv/lib/python3.5/site-packages/allauth/socialaccount/providers/oauth/client.py:168
#, python-format
msgid "No access token saved for \"%s\"."
msgstr ""

#: venv/lib/python3.5/site-packages/allauth/socialaccount/providers/oauth/client.py:188
#, python-format
msgid "No access to private resources at \"%s\"."
msgstr ""

#: venv/lib/python3.5/site-packages/allauth/templates/account/account_inactive.html:5
#: venv/lib/python3.5/site-packages/allauth/templates/account/account_inactive.html:8
msgid "Account Inactive"
msgstr "Konto inaktiv"

#: venv/lib/python3.5/site-packages/allauth/templates/account/account_inactive.html:10
msgid "This account is inactive."
msgstr "Denne konto er inaktiv"

#: venv/lib/python3.5/site-packages/allauth/templates/account/email.html:5
msgid "Account"
msgstr "Konto"

#: venv/lib/python3.5/site-packages/allauth/templates/account/email.html:8
msgid "E-mail Addresses"
msgstr "Emailadresser"

#: venv/lib/python3.5/site-packages/allauth/templates/account/email.html:10
msgid "The following e-mail addresses are associated with your account:"
msgstr "De følgende emailadresser er associeret med din konto."

#: venv/lib/python3.5/site-packages/allauth/templates/account/email.html:24
msgid "Verified"
msgstr "Verificeret"

#: venv/lib/python3.5/site-packages/allauth/templates/account/email.html:26
msgid "Unverified"
msgstr "Ikke verificeret"

#: venv/lib/python3.5/site-packages/allauth/templates/account/email.html:28
msgid "Primary"
msgstr "Primær"

#: venv/lib/python3.5/site-packages/allauth/templates/account/email.html:34
msgid "Make Primary"
msgstr "Gør til primær"

#: venv/lib/python3.5/site-packages/allauth/templates/account/email.html:35
msgid "Re-send Verification"
msgstr "Gen-send verifikation"

#: venv/lib/python3.5/site-packages/allauth/templates/account/email.html:36
#: venv/lib/python3.5/site-packages/allauth/templates/socialaccount/connections.html:35
msgid "Remove"
msgstr "Slet"

#: venv/lib/python3.5/site-packages/allauth/templates/account/email.html:43
msgid "Warning:"
msgstr "Advarsel:"

#: venv/lib/python3.5/site-packages/allauth/templates/account/email.html:43
msgid ""
"You currently do not have any e-mail address set up. You should really add "
"an e-mail address so you can receive notifications, reset your password, etc."
msgstr ""

#: venv/lib/python3.5/site-packages/allauth/templates/account/email.html:48
msgid "Add E-mail Address"
msgstr ""

#: venv/lib/python3.5/site-packages/allauth/templates/account/email.html:53
msgid "Add E-mail"
msgstr ""

#: venv/lib/python3.5/site-packages/allauth/templates/account/email.html:62
msgid "Do you really want to remove the selected e-mail address?"
msgstr ""

#: venv/lib/python3.5/site-packages/allauth/templates/account/email/email_confirmation_message.txt:1
#, python-format
msgid ""
"Hello from %(site_name)s!\n"
"\n"
"You're receiving this e-mail because user %(user_display)s at "
"%(site_domain)s has given yours as an e-mail address to connect their "
"account.\n"
"\n"
"To confirm this is correct, go to %(activate_url)s\n"
msgstr ""

#: venv/lib/python3.5/site-packages/allauth/templates/account/email/email_confirmation_message.txt:7
#, python-format
msgid ""
"Thank you from %(site_name)s!\n"
"%(site_domain)s"
msgstr ""

#: venv/lib/python3.5/site-packages/allauth/templates/account/email/email_confirmation_subject.txt:3
msgid "Please Confirm Your E-mail Address"
msgstr ""

#: venv/lib/python3.5/site-packages/allauth/templates/account/email/password_reset_key_message.txt:1
#, python-format
msgid ""
"Hello from %(site_name)s!\n"
"\n"
"You're receiving this e-mail because you or someone else has requested a "
"password for your user account at %(site_domain)s.\n"
"It can be safely ignored if you did not request a password reset. Click the "
"link below to reset your password."
msgstr ""

#: venv/lib/python3.5/site-packages/allauth/templates/account/email/password_reset_key_message.txt:8
#, python-format
msgid "In case you forgot, your username is %(username)s."
msgstr ""

#: venv/lib/python3.5/site-packages/allauth/templates/account/email/password_reset_key_message.txt:10
#, python-format
msgid ""
"Thank you for using %(site_name)s!\n"
"%(site_domain)s"
msgstr ""

#: venv/lib/python3.5/site-packages/allauth/templates/account/email/password_reset_key_subject.txt:3
msgid "Password Reset E-mail"
msgstr ""

#: venv/lib/python3.5/site-packages/allauth/templates/account/email_confirm.html:6
#: venv/lib/python3.5/site-packages/allauth/templates/account/email_confirm.html:10
msgid "Confirm E-mail Address"
msgstr ""

#: venv/lib/python3.5/site-packages/allauth/templates/account/email_confirm.html:16
#, python-format
msgid ""
"Please confirm that <a href=\"mailto:%(email)s\">%(email)s</a> is an e-mail "
"address for user %(user_display)s."
msgstr ""

#: venv/lib/python3.5/site-packages/allauth/templates/account/email_confirm.html:20
msgid "Confirm"
msgstr ""

#: venv/lib/python3.5/site-packages/allauth/templates/account/email_confirm.html:27
#, python-format
msgid ""
"This e-mail confirmation link expired or is invalid. Please <a href="
"\"%(email_url)s\">issue a new e-mail confirmation request</a>."
msgstr ""

#: venv/lib/python3.5/site-packages/allauth/templates/account/login.html:6
#: venv/lib/python3.5/site-packages/allauth/templates/account/login.html:10
#: venv/lib/python3.5/site-packages/allauth/templates/account/login.html:43
msgid "Sign In"
msgstr ""

#: venv/lib/python3.5/site-packages/allauth/templates/account/login.html:15
#, python-format
msgid ""
"Please sign in with one\n"
"of your existing third party accounts. Or, <a href=\"%(signup_url)s\">sign "
"up</a>\n"
"for a %(site_name)s account and sign in below:"
msgstr ""

#: venv/lib/python3.5/site-packages/allauth/templates/account/login.html:25
msgid "or"
msgstr ""

#: venv/lib/python3.5/site-packages/allauth/templates/account/login.html:32
#, python-format
msgid ""
"If you have not created an account yet, then please\n"
"<a href=\"%(signup_url)s\">sign up</a> first."
msgstr ""

#: venv/lib/python3.5/site-packages/allauth/templates/account/login.html:42
msgid "Forgot Password?"
msgstr ""

#: venv/lib/python3.5/site-packages/allauth/templates/account/logout.html:5
#: venv/lib/python3.5/site-packages/allauth/templates/account/logout.html:8
#: venv/lib/python3.5/site-packages/allauth/templates/account/logout.html:17
msgid "Sign Out"
msgstr ""

#: venv/lib/python3.5/site-packages/allauth/templates/account/logout.html:10
msgid "Are you sure you want to sign out?"
msgstr ""

#: venv/lib/python3.5/site-packages/allauth/templates/account/messages/cannot_delete_primary_email.txt:2
#, python-format
msgid "You cannot remove your primary e-mail address (%(email)s)."
msgstr ""

#: venv/lib/python3.5/site-packages/allauth/templates/account/messages/email_confirmation_sent.txt:2
#, python-format
msgid "Confirmation e-mail sent to %(email)s."
msgstr "Bekræftelses e-mail sendt til %(email)s."

#: venv/lib/python3.5/site-packages/allauth/templates/account/messages/email_confirmed.txt:2
#, python-format
msgid "You have confirmed %(email)s."
msgstr "Du har bekræftet %(email)s."

#: venv/lib/python3.5/site-packages/allauth/templates/account/messages/email_deleted.txt:2
#, python-format
msgid "Removed e-mail address %(email)s."
msgstr ""

#: venv/lib/python3.5/site-packages/allauth/templates/account/messages/logged_in.txt:4
#, python-format
msgid "Successfully signed in as %(name)s."
msgstr "Du er nu logget ind som %(name)s."

#: venv/lib/python3.5/site-packages/allauth/templates/account/messages/logged_out.txt:2
msgid "You have signed out."
msgstr "Du er logget ud."

#: venv/lib/python3.5/site-packages/allauth/templates/account/messages/password_changed.txt:2
msgid "Password successfully changed."
msgstr "Ændring af adgangskode lykkedes."

#: venv/lib/python3.5/site-packages/allauth/templates/account/messages/password_set.txt:2
msgid "Password successfully set."
msgstr "Adgangskoden er gemt."

#: venv/lib/python3.5/site-packages/allauth/templates/account/messages/primary_email_set.txt:2
msgid "Primary e-mail address set."
msgstr "Primær emailadresse er sat."

#: venv/lib/python3.5/site-packages/allauth/templates/account/messages/unverified_primary_email.txt:2
msgid "Your primary e-mail address must be verified."
msgstr "Din emailadresse skal være verificeret."

#: venv/lib/python3.5/site-packages/allauth/templates/account/password_change.html:5
#: venv/lib/python3.5/site-packages/allauth/templates/account/password_change.html:8
#: venv/lib/python3.5/site-packages/allauth/templates/account/password_change.html:13
#: venv/lib/python3.5/site-packages/allauth/templates/account/password_reset_from_key.html:4
#: venv/lib/python3.5/site-packages/allauth/templates/account/password_reset_from_key.html:7
#: venv/lib/python3.5/site-packages/allauth/templates/account/password_reset_from_key_done.html:4
#: venv/lib/python3.5/site-packages/allauth/templates/account/password_reset_from_key_done.html:7
msgid "Change Password"
msgstr "Ændr adgangskode"

#: venv/lib/python3.5/site-packages/allauth/templates/account/password_reset.html:6
#: venv/lib/python3.5/site-packages/allauth/templates/account/password_reset.html:10
#: venv/lib/python3.5/site-packages/allauth/templates/account/password_reset_done.html:6
#: venv/lib/python3.5/site-packages/allauth/templates/account/password_reset_done.html:9
msgid "Password Reset"
msgstr "Nulstil adgangskode"

#: venv/lib/python3.5/site-packages/allauth/templates/account/password_reset.html:15
msgid ""
"Forgotten your password? Enter your e-mail address below, and we'll send you "
"an e-mail allowing you to reset it."
msgstr ""
"Har du glemt din adgangskode? Indtast din emailadresse nedenunder og vi vil "
"sende dig en email, som gør at du kan nulstille den."

#: venv/lib/python3.5/site-packages/allauth/templates/account/password_reset.html:20
msgid "Reset My Password"
msgstr "Nulstil min adgangskode"

#: venv/lib/python3.5/site-packages/allauth/templates/account/password_reset.html:23
msgid "Please contact us if you have any trouble resetting your password."
msgstr ""
"Kontakt os venligst, hvis du har problemer med at nulstille din adgangskode."

#: venv/lib/python3.5/site-packages/allauth/templates/account/password_reset_done.html:15
msgid ""
"We have sent you an e-mail. Please contact us if you do not receive it "
"within a few minutes."
msgstr ""
"Vi har sendt dig en email. Kontakt os venligst, hvis du ikke modtagerne "
"indenfor et par minutter."

#: venv/lib/python3.5/site-packages/allauth/templates/account/password_reset_from_key.html:7
msgid "Bad Token"
msgstr ""

#: venv/lib/python3.5/site-packages/allauth/templates/account/password_reset_from_key.html:11
#, python-format
msgid ""
"The password reset link was invalid, possibly because it has already been "
"used.  Please request a <a href=\"%(passwd_reset_url)s\">new password reset</"
"a>."
msgstr ""

#: venv/lib/python3.5/site-packages/allauth/templates/account/password_reset_from_key.html:17
msgid "change password"
msgstr "Ændr adgangskode"

#: venv/lib/python3.5/site-packages/allauth/templates/account/password_reset_from_key.html:20
#: venv/lib/python3.5/site-packages/allauth/templates/account/password_reset_from_key_done.html:8
msgid "Your password is now changed."
msgstr "Din adgangskode er nu ændret."

#: venv/lib/python3.5/site-packages/allauth/templates/account/password_set.html:5
#: venv/lib/python3.5/site-packages/allauth/templates/account/password_set.html:8
#: venv/lib/python3.5/site-packages/allauth/templates/account/password_set.html:13
msgid "Set Password"
msgstr "Indstil adgangskode"

#: venv/lib/python3.5/site-packages/allauth/templates/account/signup.html:5
#: venv/lib/python3.5/site-packages/allauth/templates/socialaccount/signup.html:5
msgid "Signup"
msgstr "Tilmeld"

#: venv/lib/python3.5/site-packages/allauth/templates/account/signup.html:8
#: venv/lib/python3.5/site-packages/allauth/templates/account/signup.html:18
#: venv/lib/python3.5/site-packages/allauth/templates/socialaccount/signup.html:8
#: venv/lib/python3.5/site-packages/allauth/templates/socialaccount/signup.html:19
msgid "Sign Up"
msgstr "Tilmeld"

#: venv/lib/python3.5/site-packages/allauth/templates/account/signup.html:10
#, python-format
msgid ""
"Already have an account? Then please <a href=\"%(login_url)s\">sign in</a>."
msgstr ""

#: venv/lib/python3.5/site-packages/allauth/templates/account/signup_closed.html:5
#: venv/lib/python3.5/site-packages/allauth/templates/account/signup_closed.html:8
msgid "Sign Up Closed"
msgstr ""

#: venv/lib/python3.5/site-packages/allauth/templates/account/signup_closed.html:10
msgid "We are sorry, but the sign up is currently closed."
msgstr ""

#: venv/lib/python3.5/site-packages/allauth/templates/account/snippets/already_logged_in.html:5
msgid "Note"
msgstr ""

#: venv/lib/python3.5/site-packages/allauth/templates/account/snippets/already_logged_in.html:5
#, python-format
msgid "you are already logged in as %(user_display)s."
msgstr ""

#: venv/lib/python3.5/site-packages/allauth/templates/account/verification_sent.html:5
#: venv/lib/python3.5/site-packages/allauth/templates/account/verification_sent.html:8
#: venv/lib/python3.5/site-packages/allauth/templates/account/verified_email_required.html:5
#: venv/lib/python3.5/site-packages/allauth/templates/account/verified_email_required.html:8
msgid "Verify Your E-mail Address"
msgstr "Verificer din emailadresse"

#: venv/lib/python3.5/site-packages/allauth/templates/account/verification_sent.html:10
msgid ""
"We have sent an e-mail to you for verification. Follow the link provided to "
"finalize the signup process. Please contact us if you do not receive it "
"within a few minutes."
msgstr ""

#: venv/lib/python3.5/site-packages/allauth/templates/account/verified_email_required.html:12
msgid ""
"This part of the site requires us to verify that\n"
"you are who you claim to be. For this purpose, we require that you\n"
"verify ownership of your e-mail address. "
msgstr ""

#: venv/lib/python3.5/site-packages/allauth/templates/account/verified_email_required.html:16
msgid ""
"We have sent an e-mail to you for\n"
"verification. Please click on the link inside this e-mail. Please\n"
"contact us if you do not receive it within a few minutes."
msgstr ""

#: venv/lib/python3.5/site-packages/allauth/templates/account/verified_email_required.html:20
#, python-format
msgid ""
"<strong>Note:</strong> you can still <a href=\"%(email_url)s\">change your e-"
"mail address</a>."
msgstr ""

#: venv/lib/python3.5/site-packages/allauth/templates/openid/login.html:9
msgid "OpenID Sign In"
msgstr ""

#: venv/lib/python3.5/site-packages/allauth/templates/shopify/login.html:9
msgid "Shopify Sign In"
msgstr ""

#: venv/lib/python3.5/site-packages/allauth/templates/socialaccount/authentication_error.html:5
#: venv/lib/python3.5/site-packages/allauth/templates/socialaccount/authentication_error.html:8
msgid "Social Network Login Failure"
msgstr ""

#: venv/lib/python3.5/site-packages/allauth/templates/socialaccount/authentication_error.html:10
msgid ""
"An error occurred while attempting to login via your social network account."
msgstr ""

#: venv/lib/python3.5/site-packages/allauth/templates/socialaccount/connections.html:5
#: venv/lib/python3.5/site-packages/allauth/templates/socialaccount/connections.html:8
msgid "Account Connections"
msgstr ""

#: venv/lib/python3.5/site-packages/allauth/templates/socialaccount/connections.html:11
msgid ""
"You can sign in to your account using any of the following third party "
"accounts:"
msgstr ""

#: venv/lib/python3.5/site-packages/allauth/templates/socialaccount/connections.html:43
msgid ""
"You currently have no social network accounts connected to this account."
msgstr ""

#: venv/lib/python3.5/site-packages/allauth/templates/socialaccount/connections.html:46
msgid "Add a 3rd Party Account"
msgstr ""

#: venv/lib/python3.5/site-packages/allauth/templates/socialaccount/login_cancelled.html:5
#: venv/lib/python3.5/site-packages/allauth/templates/socialaccount/login_cancelled.html:9
msgid "Login Cancelled"
msgstr ""

#: venv/lib/python3.5/site-packages/allauth/templates/socialaccount/login_cancelled.html:13
#, python-format
msgid ""
"You decided to cancel logging in to our site using one of your existing "
"accounts. If this was a mistake, please proceed to <a href=\"%(login_url)s"
"\">sign in</a>."
msgstr ""

#: venv/lib/python3.5/site-packages/allauth/templates/socialaccount/messages/account_connected.txt:2
msgid "The social account has been connected."
msgstr ""

#: venv/lib/python3.5/site-packages/allauth/templates/socialaccount/messages/account_connected_other.txt:2
msgid "The social account is already connected to a different account."
msgstr ""

#: venv/lib/python3.5/site-packages/allauth/templates/socialaccount/messages/account_disconnected.txt:2
msgid "The social account has been disconnected."
msgstr ""

#: venv/lib/python3.5/site-packages/allauth/templates/socialaccount/signup.html:10
#, python-format
msgid ""
"You are about to use your %(provider_name)s account to login to\n"
"%(site_name)s. As a final step, please complete the following form:"
msgstr ""
